packages/replayio/src/utils/async/logAsyncOperation.ts

```diff
@@ -6,8 +6,8 @@ import {
 } from "@replay-cli/shared/async/createDeferred";
 import { statusFailed, statusPending, statusSuccess } from "@replay-cli/shared/theme";
 import { dots } from "cli-spinners";
-import { logUpdate } from "../../../../shared/src/logUpdate";
```

Comment:
Should we add a lint rule to guard against `../shared` being part of an import?

Reply:
The `resolveErrors` fails on those correctly now, see the logic related to this:
https://github.com/replayio/replay-cli/pull/597/files#diff-22552c614a4309d01e45261b51401c60c927eafcf4b6e005245f3a611b141292R64

I rechecked auto-imports and they suggest the correct imports for me - so I'm not entirely sure how this was added here. Maybe by using a different VS Code or TS Server version?

It often doesn't have to be a big issue, even when this one went through it was just "virtualized" by Rollup (put into `dist/_virtual`). It's still a problem when that happens though.

But yes, a lint rule would be great as that would give us feedback earlier.

Reply:
My VSCode still kind of struggles with this project's config. I think the "../../shared" type paths get added when you refactor/move a file around and it tries to auto-update the paths for you. (I've tried to catch them and fix them manually but..)

Reply:
Could you recheck if you are using TS 5.5? In the command palette u can find "Select TypeScript version..."

Reply:
Huh. For some reason VS Code wasn't using the workspace version.

![Screenshot 2024-07-04 at 5 38 53 PM](https://github.com/replayio/replay-cli/assets/29597/bf5cb03f-5610-420b-be88-72c68ea6b703)

Reply:
It never does by default 😢  - they claim it's for security reasons. "Select TypeScript version..." is basically memory muscle for me at this point whenever I open a new repo

Reply:
Weird bc the same value that works to "fix" this in a project setting doesn't seem to work when I put it in my user settings. And adding it to the project settings for e.g. Devtools modifies a committed VSCode settings file 🤪

Reply:
> And adding it to the project settings for e.g. Devtools modifies a committed VSCode settings file 🤪

Yep, for the long time I committed this file to repos because I believed that it would "fix" this for other contributors... but even that's not enough. They require explicit user consent each time. I never tried to put this in my user settings though.